src/games/gameitem.py

```diff
@@ -285,7 +289,11 @@ def update(self, message, old_client=None):
         # Spawn announcers: IF we had a gamestate change, show replay and hosting announcements
         if oldstate != self.state:
             if self.state == "playing":  # The delay is there because we have a 5 minutes delay in the livereplay server
-                QtCore.QTimer.singleShot(5*60000, self.announceReplay)
+                self.launched_at = message['launched_at']  # needed in Chatter update for delay icon
+                if time.time() - self.launched_at > 5*60:  # most games after client start
+                    self.announceReplay()
+                else:
+                    QtCore.QTimer().singleShot(5*60010 - 1000*(time.time() - self.launched_at), self.announceReplay)
```

Comment:
same argument as above, not to many changes at once.
the extra 0,1 second: during testing I found cases, where to singleshot seems to have come to early and the icon stayed red. with the extra time it's on the save side.